### DISCUSSION TITLE
Are there any feature-gaps?

### DISCUSSION THREAD

**Discussion by astangelo** [electric-sql/pglite#22] [General]
This is a very cool project.

I was looking for maybe a list of unimplemented Postgres features, and I didn't see any.  Are there any gaps yet identified?

**Comment by astangelo** [comment]
In particular, I'm wondering if there is full support of PGs roles.  PG's [row-level security policies](https://www.postgresql.org/docs/current/ddl-rowsecurity.html) is one of it's most useful (and most differentiating) features.  Is this fully supported?

**Comment by thruflo** [comment]
One thing is that PGLite runs in single user mode. I'm wondering how this fits with RLS. Is there an authentication context and how might credentials be provided?

With Electric specifically, we're implementing similar row-level security using [DDLX rules](https://electric-sql.com/docs/api/ddlx). We have some discussions coming up this week about the implementation path with DDLX and PGLite. It may be interesting to see what comes out of that.
